filebeat/tests/system/test_modules.py

```diff
@@ -166,7 +169,7 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
 
         cmd = [
             self.filebeat, "-systemTest",
-            "-e", "-d", "*", "-once",
```

Comment:
do we still need `-e` ?

Reply:
No, now the logs go to a file. I made this change because the home grown python framework we use tries to print the log files on a test error, if the log files do not exist (due to `-e`), then this bit throws an exception, and that's the first thing we see instead of seeing the real test failure.